packages/server/src/plugin/landingPage/default/types.ts

```diff
@@ -38,10 +38,10 @@ type InitialStateForEmbeds =
 
 export type ApolloServerPluginLandingPageDefaultBaseOptions = {
   /**
-   * By default, the landing page plugin uses the latest version of the landing
-   * page published to Apollo's CDN. If you'd like to pin the current version,
-   * pass the SHA served at
-   * https://apollo-server-landing-page.cdn.apollographql.com/_latest/version.txt
+   * By default, the landing page plugin uses the version of the landing
+   * page published to Apollo's CDN from https://github.com/apollographql/embeddable-explorer/commit/73baf89f5308d263425bbafe21a842a26123cfac.
+   * If you'd like to pin the current version, pass the SHA served at
+   * https://apollo-server-landing-page.cdn.apollographql.com/latest/version.txt
```

Comment:
I'm not sure the putting a specific commit is accurate over time here. Can we simplify this comment a bit? Something like this or maybe just a bulleted list of what's acceptable here.
> Specify the major version of the landing page to use, i.e `v2` or provide a specific commit from the [embeddable explorer](https://github.com/apollographql/embeddable-explorer) repo. We also support a `latest` tag but don't recommend using it as it's subject to major version updates.